.editorconfig

```diff
@@ -0,0 +1,31 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
```

Comment:
you can add a `max_line_length` feature as well.

Reply:
@newbazz  What should be the value of `max_line_length`.

Reply:
same goes with this.

Reply:
have a read on what this file does ;)